app/src/models/release-notes.ts

```diff
@@ -18,3 +18,6 @@ export type ReleaseSummary = {
   readonly bugfixes: ReadonlyArray<ReleaseNote>
   readonly other: ReadonlyArray<ReleaseNote>
 }
+
+export const externalContributionRe = /^(.*?)((\s?\#\d+\s?)+)(.*)(@[a-zA-Z0-9\-]+)!(.*)$/
+export const otherContributionRe = /^(.*?)((\s?\#\d+\s?)*)(.*)$/
```

Comment:
Could we please document these? I don't understand the distinction between external and other contributions.